test/functional/feature_dirsymlinks.py

```diff
@@ -6,34 +6,36 @@
 """
 
 import os
-import sys
 
-from test_framework.test_framework import BitcoinTestFramework, SkipTest
+from test_framework.test_framework import BitcoinTestFramework
 
 
 def rename_and_link(*, from_name, to_name):
     os.rename(from_name, to_name)
     os.symlink(to_name, from_name)
     assert os.path.islink(from_name) and os.path.isdir(from_name)
 
-class SymlinkTest(BitcoinTestFramework):
-    def skip_test_if_missing_module(self):
-        if sys.platform == 'win32':
-            raise SkipTest("Symlinks test skipped on Windows")
 
+class SymlinkTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
 
     def run_test(self):
+        dir_new_blocks = self.nodes[0].chain_path / "new_blocks"
```

Comment:
Any specific reason to assign these to variables?

Reply:
Sorry, no reason. I needed this for the follow-up test. Though, the test uncovered another bug (https://github.com/bitcoin/bitcoin/pull/24381#issuecomment-1044598093), so I removed it.

Happy to revert the variables as well. Lmk?

Reply:
No, if you need it for a follow-up test it's fine.